ci/aws-analytical-env/shared/meta.yml

```diff
@@ -471,8 +471,65 @@ meta:
             args:
               - -exc
               - |
-                AWS_INGESTION_INPUT_STORAGE_BUCKET="$(cat terraform-output-common-crown/outputs.json |  jq -r '.ingestion_input_storage.value.id')"
-                aws s3 --endpoint-url=https://s3-eu-west-1.amazonaws.com sync cyi-files/ s3://${AWS_INGESTION_INPUT_STORAGE_BUCKET}/${S3_FOLDER}
+                # use 25... ingestion-input-storage bucketID
+                AWS_PUBLISHED_BUCKET="SOME-BUCKET-ID"
+                aws s3 --endpoint-url=https://s3-eu-west-1.amazonaws.com sync cyi-files/ s3://${AWS_PUBLISHED_BUCKET}/${S3_FOLDER}
+
+    get-cyi-files-from-s3:
+      task: get-cyi-files-from-s3
+      attempts: 10
+      config:
+        platform: linux
+        image_resource:
+          type: docker-image
+          source:
+            repository: ((docker-awscli-ubuntu.repository))
+            version: ((docker-awscli-ubuntu.version))
+            tag: ((docker-awscli-ubuntu.version))
+        inputs:
+          - name: .aws
+        outputs:
+          - name: cyi-files-from-s3
+        params:
+          AWS_PROFILE: ci
+          AWS_SHARED_CREDENTIALS_FILE: .aws/credentials
+        run:
+          path: sh
+          args:
+            - -exc
+            - |
+              YESTERDAY="$(date +%Y-%m-%d -d 'yesterday')"
+              # use 25... ingestion-input-storage bucketID
+              SOME-BUCKET-ID="XXXXX"
+              aws s3 --endpoint-url=https://s3-eu-west-1.amazonaws.com cp --recursive s3://${SOME-BUCKET-ID}/${S3_FOLDER}/$YESTERDAY cyi-files-from-s3/.
```

Comment:
Make $YESTERDAY to ${YESTERDAY} just to keep it consistent (you might get funny behaviour in future)